naughty_strings_test.go

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package ionhash
+
+import (
+	"bufio"
+	"encoding/base64"
+	"os"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/amzn/ion-go/ion"
+)
+
+func TestNaughtyStrings(t *testing.T) {
+	t.Skip() // Skipping test until ion text reader SymbolTable() is implemented
```

Comment:
Let's create an issue for each dependency we have on ion-hash that has made us skip test. I know this is at lest the second case, we had `IsStruct` and `SymbolTable`. Make an issue for each, please.